style.css

```diff
@@ -102,28 +104,26 @@ h3 {
     text-align: center;
     font-size: 36px;
     font-weight: bolder;
-    color: #1F2937;
+    color: #1f2937;
 }
 
 .boxes {
-    background-color: #ffffff;
+    background-color: #fff;
     display: flex;
     justify-content: center;
-    height: 360px;
     gap: 64px;
+    margin: 0 auto 124px;
+    flex-wrap: wrap;
 }
 
 .boxBlock {
-    width: 160px;
+    max-width: 170px;
 }
 
 .box {
-    border: 2px solid #3882F6;
+    aspect-ratio: 1 / 1;
+    border: 4px solid #3882f6;
     border-radius: 16px;
-    outline-style: Solid;
```

Comment:
Generally `outline` is used for accessibility purposes, instead you'll want to use `border` 99% of the time.